browser/js/home/home.js

```diff
@@ -8,6 +8,21 @@ app.config(function($stateProvider) {
 	});
 });
 
-app.controller('HomeCtrl', function($scope) {
+app.factory("GetDataFactory", function($http) {
```

Comment:
I would put the app.factory in a separate file. if it's specific to the Home route, put it in home/homeFactory.js. if it's something we need to use everywhere, put it in common/factories/getDataFactory.js